source/common/network/connection_impl.cc

```diff
@@ -438,6 +450,27 @@ void ConnectionImpl::onWriteReady() {
     ASSERT(0 == rc);
     UNREFERENCED_PARAMETER(rc);
 
+#if !defined(__linux__)
```

Comment:
I believe it's true for FreeBSD and OS X. I'll just change it to #ifdef \_\_APPLE\_\_.